translate/expression/py_expression.py

```diff
@@ -30,7 +31,12 @@ def call(node):
 """
 def name(node):
     #FIXME: prob going to have type issues since logstatement is very different than node.id
-    if node.id == "print":
-        return "logStatement"
-    else:
-        return node.id      
\ No newline at end of file
+    #if node.id == "print":
+    #    return "logStatement"
+    return {"type": "name", "value": node.id}   
+
+
+def attribute(node):
```

Comment:
Please add function comment to describe what attribute means in this context.

Reply:
Just added it!